MetaMask/metamask-mobile
# feat(ramp): add orders list and list items

> **Note**
> Added interim empty state since it was not part of the designs 

## **Description**

This PR refactors the Purchases Tab into Orders, adding new components to render the sell orders and filtering at the top.

## **Manual testing steps**

1. Go to the Activity view
2. Confirm the list of Orders as seen in the screenshots

## **Screenshots/Recordings**

### **Before**

_pending_

### **After**

<img width="300" src="https://github.com/MetaMask/metamask-mobile/assets/1024246/871d367f-d248-4b7b-b795-8665f96b8d30" />
<img width="300" src="https://github.com/MetaMask/metamask-mobile/assets/1024246/439b2273-9863-46ee-bf92-6e229524424e" />
<img width="300" src="https://github.com/MetaMask/metamask-mobile/assets/1024246/a96378aa-d961-4b2f-a55e-35f64d526a47" />



## **Related issues**

_Fixes #???_

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
